web/data/passes.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
Do you think it might make sense to generate this information by using the `defaiult_passes` and `pass_manager` stuff exposed by the `calyx` crate?

Reply:
Indeed; that was one of my to-do list bullets:

> Get the pass list by querying the library, instead of needing to hand-enumerate the list in `passes.json`.